lldb/tools/lldb-dap/Protocol/ProtocolEvents.h

```diff
@@ -56,6 +57,14 @@ struct ModuleEventBody {
 llvm::json::Value toJSON(const ModuleEventBody::Reason &);
 llvm::json::Value toJSON(const ModuleEventBody &);
 
+struct InvalidatedEventBody {
+  enum Area : unsigned { eAreaAll, eAreaStacks, eAreaThreads, eAreaVariables };
+
+  std::vector<Area> areas;
```

Comment:
Should we have the fields for `threadId` and `stackFrameId` in case we add that in the future?